src/shared/lib/redirectHelpers.ts

```diff
@@ -59,6 +60,12 @@ export function handleCaseDO(){
 
 }
 
+export function handleStudyDO(){
+    const routingStore:ExtendedRouterStore = getBrowserWindow().routingStore;
+    const currentQuery = routingStore.location.query;
```

Comment:
seems to me first param (currentQuery) should be {} and third param (clear) should be false.  We're maintaining the current query so we don't want to clear it and we don't need to pass anything in as newParams.